mypy.ini

```diff
@@ -1,5 +1,35 @@
 [mypy]
-ignore_missing_imports = True
-# required to support namespace packages
-# https://github.com/python/mypy/issues/14057
+# Is the project well-typed?
+strict = False
+
+# Early opt-in even when strict = False
+warn_unused_ignores = True
+warn_redundant_casts = True
+enable_error_code = ignore-without-code
+
+# Support namespace packages per https://github.com/python/mypy/issues/14057
 explicit_package_bases = True
+
+# Disable overload-overlap due to many false-positives
+disable_error_code = overload-overlap
+
+exclude = (?x)(
+    # upstream
+	^build/
+	| ^.tox/
+ 
+ 	# local
+	)
+
+# - distutils._modified has different errors on Python 3.8 [import-untyped], on Python 3.9+ [import-not-found]
+# - All jaraco modules are still untyped
+# - wheel appears to be untyped
+[mypy-distutils._modified,jaraco.*,wheel.*]
+ignore_missing_imports = True
+
+# Even when excluding a module, import issues can show up due to following import
+# https://github.com/python/mypy/issues/11936#issuecomment-1466764006
+# Add your excluded modules that still produce import errors here
+# [mypy-a_module.*]
+# follow_imports = silent
+# silent => ignore errors when following imports
```

Comment:
Probably this kind of guidance should go in the skeleton docs (gh-pages branch) with maybe a one-line link at the top of the file.

Reply:
Sure! Not sure exactly where this goes in https://github.com/jaraco/skeleton/blob/gh-pages/index.md though. But it sounds like that can be done in parallel, so I'll remove this section from this file now.
```suggestion
```

Reply:
I thought maybe there was already a relevant section, but I don't see one. I think it should go under Packaging Conventions after Running Tests, something like Type Checking

## Packaging Conventions

### Running Tests

...

### Type Checking

...

### Continuous Integration

...